docs/deployment/resource-providers/mesos.md

```diff
@@ -26,226 +26,242 @@ under the License.
 * This will be replaced by the TOC
 {:toc}
 
-## Background
+## Getting Started
 
-The Mesos implementation consists of two components: The Application Master and
-the Worker. The workers are simple TaskManagers which are parameterized by the environment
-set up by the application master. The most sophisticated component of the Mesos
-implementation is the application master. The application master currently hosts
-the following components:
+This *Getting Started* section guides you through setting up a fully functional Flink Cluster on Mesos.
 
-### Mesos Scheduler
+### Introduction
 
-The scheduler is responsible for registering the framework with Mesos,
-requesting resources, and launching worker nodes. The scheduler continuously
-needs to report back to Mesos to ensure the framework is in a healthy state. To
-verify the health of the cluster, the scheduler monitors the spawned workers and
-marks them as failed and restarts them if necessary.
+[Apache Mesos](http://mesos.apache.org/) is another resource provider supported by 
+Apache Flink. Flink utilizes the worker's provided by Mesos to run its TaskManagers.
+Apache Flink provides the script `bin/mesos-appmaster.sh` to initiate the Flink 
+on Mesos cluster.
 
-Flink's Mesos scheduler itself is currently not highly available. However, it
-persists all necessary information about its state (e.g. configuration, list of
-workers) in Zookeeper. In the presence of a failure, it relies on an external
-system to bring up a new scheduler. The scheduler will then register with Mesos
-again and go through the reconciliation phase. In the reconciliation phase, the
-scheduler receives a list of running workers nodes. It matches these against the
-recovered information from Zookeeper and makes sure to bring back the cluster in
-the state before the failure.
+### Preparation
 
-### Artifact Server
+Flink on Mesos expects a Mesos cluster to be around. It also requires the Flink binaries being 
+deployed. Additionally, Hadoop needs to be installed on the very same machine.
```

Comment:
Is it also good enough if we add the Hadoop dependencies to Flink? Then the users don't have to install Hadoop. Maybe we could say that Hadoop needs to be installed or you have to provide the Hadoop dependencies.

Reply:
Yes, using the bundled hadoop uber jar would work as well. I added the  `#supported-hadoop-versions` to the `Flink on Mesos Reference` section at the end of this page. I didn't mention it here since we claim the `HADOOP_CLASSPATH` to be the preferred approach.

Reply:
Fair enough.